contracts/connection-router/src/msg.rs

```diff
@@ -17,12 +17,12 @@ pub enum ExecuteMsg {
      */
     // Registers a new chain with the router
     RegisterChain {
-        chain: String,
+        chain: ChainName,
```

Comment:
To keep this PR simple, let's just change the service registry (and anything else we need to change because we changed the service registry). Let's leave the connection-router alone.

Reply:
Why leaving the router alone makes this PR simpler?